config/default.js

```diff
@@ -41,7 +41,16 @@ module.exports = {
       logging: true,
       database: 'streetmix_dev',
       host: process.env.PGHOST || '127.0.0.1',
-      port: process.env.PGPORT || 5432
+      port: process.env.PGPORT || 5432,
+      // surely there is a way to inherit these defaults to other config files?
```

Comment:
Not sure what this comment means, do we mean exporting the values, or that the config object should inherit properties from default? If the latter, I believe that's already happening.